packages/tui/src/screens/chat.tsx

```
@@ -265,6 +370,264 @@ export function Chat({
+                                        (statusBlock.content === 'Working...' ||
+                                            statusBlock.content === 'Thinking...' ||
+                                            statusBlock.content.startsWith('Rate limit hit'))
+                                    ) {
+                                        statusBlock.content = event.message || 'Working...'
+                                    } else if (event.message) {
+                                        blocks.push({ type: 'text', content: event.message })
+                                    }
+                                    break
+                                }
+                                case 'StreamChunk':
+                                    const lastBlock = blocks[blocks.length - 1]
```

Biome (2.5.1)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

(lint/correctness/noSwitchDeclarations)